tracer/test/test-applications/integrations/Samples.OpenTracing/Samples.OpenTracing.csproj

```diff
@@ -6,4 +6,9 @@
     <ProjectReference Include="..\..\..\..\src\Datadog.Trace.OpenTracing\Datadog.Trace.OpenTracing.csproj" />
     <PackageReference Include="FluentAssertions" Version="6.4.0" />
   </ItemGroup>
+
```

Comment:
[ERR] CompileSamples: C:\Commonfolder\shared\repos\dd-trace-5\tracer\test\test-applications\integrations\Samples.OpenTracing\Program.cs(107,9): error CS0012: The type 'HttpResponseMessage' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.